test/_initFixture.js

```diff
@@ -29,7 +29,7 @@ export default function initFixture(fixturePath, callback) {
   }, err => {
     if (err) return callback(err);
     process.chdir(testDir);
-    child.execSync("git init . && git add -A && git commit -m 'Init commit'");
+    child.execSync('git init . && git add -A && git commit -m "Init commit"');
```

Comment:
Ya i saw, but this actually fixes a windows issue, believe or not.

Basically on windows commit message MUST be in double quotes, not single like it was before. So i turned ths string to have single on the outside so i could get double inside.

If you like i could change it to a es6 template literal string. But need to keep double quote around the commit message for windows compatability.